xml/ay_bigfile.xml

```diff
@@ -8984,64 +8984,818 @@ config:type="boolean"&gt;true&lt;/keep_install_network&gt;</screen>
-    users. Additional options are available for a more customized user
-    account management. The data in
-    <filename>/etc/default/useradd</filename> is used to determine the home
-    directory of the user to be created plus other parameters.
-   </para>
-   <note>
-     <title>Users set up during the first stage</title>
-     <para>
-      Starting with &productname; 12 SP1, users are set up during the first
-      stage (in previous versions it happened in the second one). So with a
-      minimal profile, you can set up a usable system running only the first
-      stage.
+    <informaltable>
+     <tgroup cols="3">
+      <thead>
+       <row>
+        <entry>
+         <para>
+          Attribute
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Values
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Description
+         </para>
+        </entry>
+       </row>
+      </thead>
+      <tbody>
+       <row>
+        <entry>
+         <para>
+          <literal>username</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Text
+         </para>
+<screen>&lt;username&gt;lukesw&lt;/username&gt;</screen>
+        </entry>
+        <entry>
+         <para>
+          Required. It should be a valid username. Check <literal>man 8 useradd</literal>
+          if you're not sure.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>fullname</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Text
+         </para>
+<screen>&lt;fullname&gt;Tux Torvalds&lt;/fullname&gt;</screen>
+        </entry>
+        <entry>
+         <para>
+          Optional. User's fullname.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>forename</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Text
+         </para>
+<screen>&lt;forname&gt;Tux&lt;/forename&gt;</screen>
+        </entry>
+        <entry>
+         <para>
+          Optional. User's forename.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>surname</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Text
+         </para>
+<screen>&lt;surname&gt;Skywalkwer&lt;/surname&gt;</screen>
+        </entry>
+        <entry>
+         <para>
+          Optional. User's surname.
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          <literal>uid</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Number
+         </para>
+<screen>&lt;uid&gt;1001&lt;/uid&gt;</screen>
+        </entry>
+        <entry>
+         <para>
+          Optional. User ID. It must be a unique and non-negative number.
```

Comment:
Good point. We have had an ugly error due equal UIDs  which has costed days to find out .-)